internal/manager/task_stash_box_tag.go

```diff
@@ -51,7 +78,14 @@ func (t *StashBoxPerformerTagTask) stashBoxPerformerTag(ctx context.Context) {
 	if t.refresh {
 		var performerID string
 		txnErr := txn.WithReadTxn(ctx, instance.Repository, func(ctx context.Context) error {
-			stashids, _ := instance.Repository.Performer.GetStashIDs(ctx, t.performer.ID)
+			if !t.performer.StashIDs.Loaded() {
+				err = t.performer.LoadStashIDs(ctx, instance.Repository.Performer)
+				if err != nil {
+					return err
+				}
+			}
+			stashids := t.performer.StashIDs.List()
```

Comment:
The StashIDs are now passed in to the task, but check to load them just in case.